Expensify/App
# Add asyncOpenURL wrapper

cc @chiragsalian @roryabraham 

### Fixed Issues
$ https://github.com/Expensify/Expensify.cash/issues/3615

### Tests
1. Send a `Request Money` request from Account A to Account B (both accounts must have PayPal.me username set in `Settings > Payments`)
2. Open Safari and log in to Account B.
3. From Account B, click Pay and from the Dropdown select Pay with `PayPal.me` and then confirm the payment. 
4. A new window should open and 
4. Verify that you are redirected to Paypal.me and the handle is that of the user you owe and amount matches.

### QA Steps
Steps above.

### Tested On


### Screenshots

#### Web

https://user-images.githubusercontent.com/22219519/123491035-f7ab9e00-d5d2-11eb-9b5c-fd553da32b4d.mov

#### Mobile Web

https://user-images.githubusercontent.com/22219519/123680754-4bea9400-d806-11eb-8a0d-aa479002a0f0.mov

#### Desktop

https://user-images.githubusercontent.com/22219519/123680775-5147de80-d806-11eb-8c77-697ac3baf754.mov

#### iOS

https://user-images.githubusercontent.com/22219519/123680958-776d7e80-d806-11eb-9f3b-ed1a992bd014.mov

#### Android

https://user-images.githubusercontent.com/22219519/123680814-5c027380-d806-11eb-9d39-4714c7e869cc.mov
